### PR TITLE
RFC 74: Add testdriver.js support for WebDriver delete_all_cookies command

### PR DESCRIPTION
**Pull request by miketaylr**
PR to add support for `delete_all_cookies` here: https://github.com/web-platform-tests/wpt/pull/26723

### CODE REVIEW

**Comment by annevk** [comment]
WebDriver doesn't seem particularly detailed about how it serializes cookie values. That seems rather important to address first given that JSON doesn't have a native representation for byte sequences.

**Comment by miketaylr** [comment]
> at least before adding anything apart from "delete all"

This is the one bit I'm actually interested in right now, so I'm happy to reduce the scope of the RFC to land this without blocking on the rest.

**Comment by annevk** [comment]
Sounds good, I filed https://github.com/w3c/webdriver/issues/1562 on WebDriver for the serialization issue.

**Comment by jgraham** [comment]
I think this makes sense.

**Comment by miketaylr** [comment]
> Sounds good, I filed [w3c/webdriver#1562](https://github.com/w3c/webdriver/issues/1562) on WebDriver for the serialization issue.

Thanks. I've updated the RFC as well.

File: rfcs/testdriver_cookie_methods.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+and others are prone to race conditions, or [don't cover all use cases](https://github.com/web-platform-tests/wpt/blob/d1aaf685cfb02e0350701550afe4146d555f2461/cookies/resources/cookie.py#L13-L17).
+
+Rather than re-write existing support code to fix these issues, this RFC aims to add support for the WebDriver `delete_all_cookies` command. An PR exists at https://github.com/web-platform-tests/wpt/pull/26723`, as it addresses a known issue of [flaky leftover state](https://github.com/web-platform-tests/wpt/issues/26707).
+
+Future RFCs may propose adding the rest of the cookie commands, once [cookie serialization is defined by WebDriver](https://github.com/w3c/webdriver/issues/1562). That will allow writing cookie tests independent of HTTP or DOM cookie APIs (i.e., `Set-Cookie`, `document.cookie`, `CookieStore`).
+
+## Risks
+
+This increases the API surface of testdriver.js, so it's more code to maintain or reason about. In addition, the developer experience of running local cookie tests will be less convenient (requiring `wpt run` instead of just hitting refresh).
```
Comment (stephenmcgruer) [inline rfcs/testdriver_cookie_methods.md:18]: As just a side-note; if you running a single test under `wpt run` we do keep the test window open after the test finishes and I believe refreshing the test should work fine. (Unless there's weird stateful things happening I don't know about).

**Review by stephenmcgruer — APPROVED** [review]
LGTM.

**Comment by miketaylr** [comment]
It's been 7 days with no objections, shall we merge this?

**Pull request merged**